src/devices/src/virtio/vsock/tcp.rs

```diff
@@ -1,15 +1,20 @@
 use std::collections::HashMap;
-use std::net::{Ipv4Addr, SocketAddrV4};
+use std::net::{Ipv4Addr, SocketAddrV4, SocketAddrV6};
 use std::num::Wrapping;
 use std::os::fd::{FromRawFd, OwnedFd};
 use std::os::unix::io::{AsRawFd, RawFd};
 use std::sync::{Arc, Mutex};
 
+use libc::AF_INET;
+#[cfg(target_os = "linux")]
+use libc::EINVAL;
+#[cfg(target_os = "macos")]
+use libc::EINVAL;
```

Comment:
nit (nonblocking): do we need the differentiation here? We only support these two OSes, but would it be a poor assumption for us to just import `libc::EINVAL`?